MapboxNavigation.xcodeproj/xcshareddata/xcschemes/Bench.xcscheme

```diff
@@ -1,97 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Scheme
```

Comment:
Should this scheme be shared as part of the new project?

Reply:
Yeah, nice catch.